glide.yaml

```diff
@@ -16,6 +16,10 @@ import:
   version: v0.0.1
 - package: github.com/spf13/pflag
   version: v1.0.0
+- package: github.com/openshift/api
+- package: github.com/docker/docker
```

Comment:
Why docker? We don't import it anywhere.

Reply:
To further your comment, this is added to fix the Glide issue: https://github.com/redhat-developer/odo/issues/377 OpenShift uses docker/docker

Reply:
This fixes the issue by specifying that we're going to use github.com/openshift/api as well as github.com/docker/docker within Glide. After going through: https://github.com/Masterminds/glide/issues/945 the solution was to simply add it to `glide.yaml` and it'll fix itself.

Reply:
Can you please add a comment explaining why we have it in glide.yml even tho we are not importing it anywhere?